generated/nidigital/nidigital/session.py

```diff
@@ -2749,7 +3233,7 @@ def abort_keep_alive(self):
     def commit(self):
         r'''commit
 
-        TBD
+        Applies all previously configured pin levels, termination modes, clocks, triggers, and pattern timing to a digital pattern instrument. If you do not call the commit method, then the _initiate method or the _burst_pattern method will implicitly call this method for you. Calling this method moves the session from the Uncommitted state to the Committed state.
```

Comment:
The source metadata refers to niDigital_Initiate and niDigital_BurstPattern, and those strings are correctly replaced with hyperlinks to the respective methods on the Session in the generated HTML documentation.  I suspect this is a bug in how the generator for the doc strings.  I am reluctant to manually fixup the files in the 'generated' folder, because future builds will overwrite it and all future PRs will have to deal with undoing that, so I'd rather create an issue to track it.

Reply:
Looked at this a bit closely..

`_burst_pattern` can be fixed by replacing `niDigital_BurstPattern` with `niDigital_FancyBurstPattern` in the metadata

`_initiate` is a known limitation in code generator (#489). The workaround for now would be to replace `niDigital_Initiate` with `initiate` (Although it would mean the hyperlinking in RST won't happen)

Reply:
I replaced `niDigital_BurstPattern` with `niDigital_FancyBurstPattern`.

Reply:
I also replaced `niDigital_Initiate` with `initiate`.